transforms/PathologyLabelSoften.py

```diff
@@ -91,30 +109,50 @@ def _create_soft_labels(self, labels: np.ndarray) -> np.ndarray:
         return labels
 
     def _smooth(self, label: np.ndarray) -> np.ndarray:
-        "Apply mean filtering to 2D numpy array"
+        """Apply median smooth to 2d numpy array"""
         label = convert_to_tensor(label)
         label = label.unsqueeze(0).unsqueeze(0).float()
         ks = 3
-        mean_kernel = torch.ones(1, 1, ks, ks) / ks**2
-        label = F.conv2d(label, mean_kernel, padding=ks // 2)
+        median_kernel = torch.ones(1, 1, ks, ks) / ks**2
```

Comment:
I've referred to this as median filtering in the past, but this was wrong. A median filter is implemented differently. This is mean filtering.